presentation/common/src/test/Update.test.ts

```diff
@@ -15,6 +15,7 @@ const testNode: Node = {
     instanceKeys: [],
     pathFromRoot: [],
     type: StandardNodeTypes.ECInstancesNode,
+    instanceKeysSelectQuery: { query: "", bindings: [] },
```

Comment:
`instanceKeysSelectQuery` should be optional and these should be unnecessary

Reply:
Removed.